influxdb3_write/src/write_buffer/metrics.rs

```diff
@@ -4,37 +4,51 @@ use metric::{Metric, Registry, U64Counter};
 
 #[derive(Debug)]
 pub(super) struct WriteMetrics {
-    write_lines_total: Metric<U64Counter>,
-    write_bytes_total: Metric<U64Counter>,
+    write_lines_count: Metric<U64Counter>,
+    write_lines_rejected_count: Metric<U64Counter>,
+    write_bytes_count: Metric<U64Counter>,
 }
 
-pub(super) const WRITE_LINES_TOTAL_NAME: &str = "influxdb3_write_lines_total";
-pub(super) const WRITE_BYTES_TOTAL_NAME: &str = "influxdb3_write_bytes_total";
+pub(super) const WRITE_LINES_METRIC_NAME: &str = "influxdb3_write_lines";
+pub(super) const WRITE_LINES_REJECTED_METRIC_NAME: &str = "influxdb3_write_lines_rejected";
```

Comment:
Should end in `_total`

Reply:
I mentioned this in the PR description, but our Prometheus exporter adds a "_total" to the end of metric names that use a counter on export, which is why I removed it from the names here.

Reply:
Oh sorry, I should have read that closer!